tests/ui/editions/edition-keywords-2018-2015-2.rs

```diff
@@ -0,0 +1,5 @@
+//@ edition:2018
+
+fn main() {
+    let _recovery_witness: () = 0; //~ ERROR mismatched types
+}
```

Comment:
The point of this was to ensure that the other test would continue emitting errors after the previous errors, so it is not needed as its own file.